stable/spinnaker/templates/_helpers.tpl

```diff
@@ -55,3 +55,14 @@ Redis base URL for Spinnaker
 {{- printf "redis://%s:%s" .Values.redis.external.host (.Values.redis.external.port | toString) -}}
 {{- end }}
 {{- end }}
+
+{{/*
+Create name of kubeconfig file to use when setting up kubernetes provider
+*/}}
+{{- define "spinnaker.kubeconfig" -}}
+{{- if .Values.kubeconfig.encryptedKubeconfig }}
+{{- printf .Values.kubeconfig.encryptedKubeconfig | toString -}}
```

Comment:
@bnutt helm chart install is failing because we specify `kubeconfig` here instead of `kubeConfig` (as defined in values.yaml - https://github.com/helm/charts/pull/16473/files#diff-13a5d824f78f72778667d929a7c83c4dL114)